src/App.js

```diff
@@ -135,13 +135,13 @@ class App extends Component {
 
   changeLanguage = (lang) => {
     tasmotaGUILanguages.sort((a, b) => {
-      return allMessages[lang]['source'][a.name].localeCompare(
-        allMessages[lang]['source'][b.name]
+      return (allMessages[lang]['source'][a.name] || a.name).localeCompare(
+          (allMessages[lang]['source'][b.name] || b.name)
```

Comment:
I don't think we need to do this in the code of the application. What needs to be implemented are proper test scenarios to find out whether there is a translation with a missing key.
`npm run test` - that is needed. This should tell us that some translations are incomplete.

Have a look at my conversation with @drhideg here: #197

Reply:
Have a look here: https://github.com/benzino77/tasmocompiler/blob/da5c38436aa36d6ff4677d9121a44e391d5f506d/src/locales/languages.js#L71-L78

I've already addressed the situation that someone has forgot about some translation. Completely forget about that ;)